.buildkite/deploy/kubernetes/deploy-k8s-pipeline.yml

```diff
@@ -1,5 +1,65 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
+env:
+  IMAGE_UBUNTU_X86_64: "family/platform-ingest-beats-ubuntu-2204"
+  GCP_HI_PERF_MACHINE_TYPE: "c2d-highcpu-16"
+
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+  - label: "Checks"
+    command: ".buildkite/deploy/kubernetes/scripts/make.sh"
```

Comment:
https://github.com/elastic/beats/pull/38175 could be another approach. IMO,  the linting/checks were moved to GitHub and this was a leftover.

Reply:
Team decided to keep the steps in Buildkite